cloud.foundry.cli/src/main/java/cloud/foundry/cli/crosscutting/logging/Log.java

```diff
@@ -0,0 +1,125 @@
+package cloud.foundry.cli.crosscutting.logging;
+
+import java.util.logging.Level;
+
+/**
+ * The central logging module, providing a standardized interface for all logging activities in the project.
+ * By using this module instead of using custom logging facilities (like System.{out,err}.print*, for instance), a
+ * consistent style is ensured and log messages can be filtered properly.
+ * The interface is similar to Python's logging module, which provides a simple-to-use yet powerful API.
+ */
+public class Log {
+    // we can internally use the Java logging facilities
+    private static final java.util.logging.Logger logger;
+
+    // initialize and configure logger initially
+    static {
+        logger = java.util.logging.Logger.getLogger("cfctl");
```

Comment:
The logger name could be declared in a private constant field

Reply:
True. It's just used in one place, but maybe one needs the value for unit testing, there's no reason not to add a public constant attribute.